decidim-core/app/views/decidim/participatory_processes/_hero_process.html.erb

```diff
@@ -0,0 +1,27 @@
+<article class="card card--full card--process">
+  <div class="row collapse card--process__row">
+    <div class="columns mediumlarge-8 large-6 card--process__column">
+      <div class="card__content">
+        <%= link_to participatory_process_path(hero_process), class: "card__link" do %>
+          <h2 class="card__title"><%= translated_attribute hero_process.title %></h2>
+        <% end %>
+        <%= truncate(translated_attribute(hero_process.description).html_safe, length: 630, separator: ' ') %>
```

Comment:
By using this syntax, given `html_safe` is a view's concern, you're forced to explicitly tell to the reader what's happening. Otherwise, by trusting `html_safe`, you're assuming the string will be marked as safe somewhere in the chain instead (which makes no sense at all).

Also, I'd stick with rails (erb?) conventions whenever possible.